script/01_DeployCLPositionDescriptorOffchain.s.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
+
+import "forge-std/Script.sol";
+import {BaseScript} from "./BaseScript.sol";
+import {CLPositionDescriptorOffChain} from "../src/pool-cl/CLPositionDescriptorOffChain.sol";
+
+/**
+ * forge script --sig 'run(uint256)' script/01_DeployCLPositionDescriptor.s.sol:DeployCLPositionDescriptorOffChainScript <baseTokenURI> -vvv \
+ *     --rpc-url $RPC_URL \
```

Comment:
should be  --sig 'run(string)' ?

Reply:
script/01_DeployCLPositionDescriptor.s.sol -> script/01_DeployCLPositionDescriptorOffchain.s.sol

Reply:
<optional> would prefer if we can read from the config though, so future devs knows that params and can deploy this easily